data/fields/crossing.json

```diff
@@ -1,5 +1,17 @@
 {
     "key": "crossing",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "options": [
+        "traffic_signals",
+        "uncontrolled",
+        "unmarked"
+    ],
+    "strings": {
+        "options": {
+            "traffic_signals": "Crossing With Traffic Signals",
+            "uncontrolled": "Only Road Markings",
+            "unmarked": "No Road Markings or Traffic Lights"
```

Comment:
These three strings are inconsistent with each other. Both “traffic signals” and “traffic lights” are correct in American English, but we should stick to one. “Crossing” is redundant in this field.

Reply:
Should `no` get a string too? (It would only make sense on a node.)

Reply:
I've fixed the wording.

> Maybe `crossing=no` should be a separate preset?

I haven't included https://wiki.openstreetmap.org/wiki/Tag:crossing%3Dno in this field, as it is intended to specify the type of a crossing, where the `no` option doesn't make sense.
`crossing=no` would indeed  be useful as a standalone preset and as dedicated field to be added to presets like the `highway=traffic_signals`.